tailwind.config.js

```diff
@@ -228,6 +228,9 @@ module.exports = {
       backgroundImage: {
         "gradient-radial": "radial-gradient(var(--tw-gradient-stops))",
       },
+      borderWidth: {
```

Comment:
@isabensusan @OgDev-01, the design required a 1px border so I've added it to our tailwind classes. It's available as `border-b-1`.